storage/files.js

```diff
@@ -72,326 +68,281 @@ function listFiles (name, callback) {
  * @param {function} cb The callback function.
  */
-function deleteFile (name, fileName, callback) {
-  if (!name) {
-    return callback(new Error('"name" is required!'));
-  } else if (!fileName) {
-    return callback(new Error('"fileName" is required!'));
-  }
-
-  var bucket = storage.bucket(name);
-  var file = bucket.file(fileName);
+function deleteFile (options, callback) {
+  var file = storage.bucket(options.bucket).file(options.file);
 
   file.delete(function (err) {
     if (err) {
       return callback(err);
     }
 
-    console.log('Deleted file: %s', fileName);
+    console.log('Deleted gs://%s/%s', options.bucket, options.file);
     return callback(null);
   });
 }
-// [END delete]
+// [END delete_file]
 
-// [START metadata]
+// [START get_metadata]
 /**
  * Get a file's metadata.
  *
- * @param {string} name The name of the bucket.
- * @param {string} fileName The name of the file.
+ * @param {object} options Configuration options.
+ * @param {string} options.bucket The name of the bucket.
+ * @param {string} options.file The name of the file.
  * @param {function} cb The callback function.
  */
-function getMetadata (name, fileName, callback) {
-  if (!name) {
-    return callback(new Error('"name" is required!'));
-  } else if (!fileName) {
-    return callback(new Error('"fileName" is required!'));
-  }
-
-  var bucket = storage.bucket(name);
-  var file = bucket.file(fileName);
+function getMetadata (options, callback) {
+  var file = storage.bucket(options.bucket).file(options.file);
 
   file.getMetadata(function (err, metadata) {
     if (err) {
       return callback(err);
     }
 
-    console.log('Got metadata for file: %s', fileName);
+    console.log('Got metadata for gs://%s/%s', options.bucket, options.file);
     return callback(null, metadata);
   });
 }
-// [END metadata]
+// [END get_metadata]
 
 // [START public]
 /**
  * Make a file public.
  *
- * @param {string} name The name of the bucket.
- * @param {string} fileName The name of the file to make public.
+ * @param {object} options Configuration options.
+ * @param {string} options.bucket The name of the bucket.
+ * @param {string} options.file The name of the file to make public.
  * @param {function} cb The callback function.
  */
-function makePublic (name, fileName, callback) {
-  if (!name) {
-    return callback(new Error('"name" is required!'));
-  } else if (!fileName) {
-    return callback(new Error('"fileName" is required!'));
-  }
+function makePublic (options, callback) {
+  var file = storage.bucket(options.bucket).file(options.file);
 
-  var bucket = storage.bucket(name);
-  var file = bucket.file(fileName);
-
-  file.makePublic(function (err, apiResponse) {
+  file.makePublic(function (err) {
     if (err) {
       return callback(err);
     }
 
-    console.log('Made %s public!', fileName);
-    return callback(null, apiResponse);
+    console.log('Made gs://%s/%s public!', options.bucket, options.file);
+    return callback(null);
```

Comment:
Is this null necessary? Can we just do `callback()`, or is that not clear enough?